pkg/protocol/header_test.go

```diff
@@ -264,6 +271,7 @@ func TestResponseHeaderDel(t *testing.T) {
 	h.Del(consts.HeaderServer)
 	h.Del("content-length")
 	h.Del("set-cookie")
+	h.DelBytes([]byte("content-encoding"))
```

Comment:
h.DelBytes -> h.Del? Check the function title, `TestResponseHeaderDel ` is for the ut of response header .Del method?